packages/gatsby/src/utils/webpack.config.js

```diff
@@ -287,6 +287,17 @@ module.exports = async (
         ])
         break
       }
+      case `develop-html`:
+      case `build-html`: {
```

Comment:
do we need `build-ssr` here?

Reply:
This builds on top of feat/page-renderer-split where build-ssr is gone and part of build-html